precompiles/delegation/IDelegation.sol

```diff
@@ -52,5 +52,29 @@ interface IDelegation {
         bytes memory operatorAddr,
         uint256 opAmount
     ) external returns (bool success);
+
+    /// TRANSACTIONS
```

Comment:
This line is superfluous; please remove.

Reply:
Thanks, I will remove them.